bin/build-utils.js

```diff
@@ -29,8 +29,11 @@ function writeFile(filename, contents) {
 }
 
 function doUglify(pkgName, code, prepend, fileOut) {
-  var miniCode = prepend + terser.minify(code, { output: { ascii_only: true }}).code;
-  return writeFile(addPath(pkgName, fileOut), miniCode);
+  const filename = fileOut.replace(/.*\//, '');
```

Comment:
one of the features of source maps is being able to click the file and line number in an ide and directly open the right file and cursor position, but this requires the relative path prefixed to the filenames, probably (not sure) also a space between the prefix and filename likse so  _setup@  project-folder/filename.js

Reply:
I don't have an IDE to test this on, but the current format matches what Firefox generates.  Happy to change to other formats in this PR or in future if helpful.

Reply:
I'm guessing following the NodeJS format would be most compatible: https://nodejs.org/docs/latest/api/errors.html#errorstack

Reply:
i am not sure how the nodejs error stack format would work in the the terminal output unless i missed a section in the docu. i would not overcomplicate it and just checked vscode which seems to be the vast majority these days and it works as long as there is a space after the @ and the path is relative to the repo. eg. :

_setup@ src/main.js:3:2


this also seem the most human readable to me, what do you think?